modules/mod_languages/tmpl/default.php

```diff
@@ -10,71 +10,72 @@
 defined('_JEXEC') or die;
 
 use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
 use Joomla\CMS\Uri\Uri;
 
 /** @var Joomla\CMS\WebAsset\WebAssetManager $wa */
 $wa = $app->getDocument()->getWebAssetManager();
 $wa->registerAndUseStyle('mod_languages', 'mod_languages/template.css');
-
 ?>
-<div class="mod-languages">
+<div class="mod-languages" aria-describedby="language_picker_des">
+	<p class="visually-hidden" id="language_picker_des"><?php echo Text::_('MOD_LANGUAGES_DESC'); ?></p>
```

Comment:
```suggestion
	<p class="visually-hidden"><?php echo Text::_('MOD_LANGUAGES_DESC'); ?></p>
```
after above you dont need the id